vpm.json

```diff
@@ -6,49 +6,114 @@
   "packages": {
     "jp.sh0uroom.imageloader": {
       "versions": {
-          "1.0.1": {
+        "1.0.1": {
           "name": "jp.sh0uroom.imageloader",
-          "version": "1.0.1",
           "displayName": "SimpleImageLoader",
+          "version": "1.0.1",
           "description": "Tools to load images from URLs.",
+          "gitDependencies": {},
+          "vpmDependencies": {
+            "com.vrchat.worlds": "3.4.1"
```

Comment:
This may force user use VRCSDK 3.4.1 with Unity 2019 but this is not bug in VCC / VPM Client, but problem of your package.
Package should allow more VRCSDK versions with `>= 3.4.1` or `^3.4.1` in package.json of your package